skiko/src/jvmTest/kotlin/org/jetbrains/skiko/FrameLimiterTest.kt

```diff
@@ -1,274 +1,274 @@
-
-        runBlocking(Dispatchers.IO) {
-            repeat(50000) {
-                frameLimiter.awaitNextFrame()
-            }
-        }
-
-        scope.cancel()
-    }
-
-    @Test(timeout = 30000)
-    fun `multiple multithreaded awaiters`() {
-        val scope = CoroutineScope(Dispatchers.IO)
-        val frameLimiter = FrameLimiter(scope, { 0 }, nanoTime = System::nanoTime)
-
-        runBlocking(Dispatchers.IO) {
-            repeat(3) {
-                launch {
-                    repeat(50000) {
-                        frameLimiter.awaitNextFrame()
-                        yield()
-                        yield()
-                        yield()
-                    }
-                }
-            }
-        }
-
-        scope.cancel()
-    }
-
-    private inline fun <reified T : Throwable> assertThrow(body: () -> Unit) {
-        var actualE: Throwable? = null
-        try {
-            body()
-        } catch (e: Throwable) {
-            actualE = e
-        }
-        assertTrue("Actual ${actualE?.javaClass}, expected ${T::class.java}", actualE is T)
-    }
-
-    private fun frameLimiterTest(
-        frameLimitMillis: Long,
-        delayPrecisionMillis: Long,
-        block: suspend TestCoroutineScope.(FrameLimiter) -> Unit
-    ) {
-        runFrameTest(
-            delayPrecisionMillis = delayPrecisionMillis
-        ) {
-            val scope = CoroutineScope(coroutineContext + Job())
-            val limiter = FrameLimiter(
-                this,
-                frameMillis = { frameLimitMillis },
-                nanoTime = { currentTime * 1_000_000 }
-            )
-            block(limiter)
-            scope.cancel()
-        }
-    }
-
-    private fun runFrameTest(
-        delayPrecisionMillis: Long,
-        block: suspend TestCoroutineScope.() -> Unit
-    ) = runBlockingTest{
-        val dispatcher = NonpreciseTestCoroutineDispatcher(delayPrecisionMillis)
-        dispatcher.pauseDispatcher()
-        val scope = TestCoroutineScope(dispatcher)
-        scope.launch {
-            scope.block()
-        }
-        dispatcher.advanceUntilIdle()
-    }
-
-    @OptIn(InternalCoroutinesApi::class)
-    private class NonpreciseTestCoroutineDispatcher(
-        private val delayPrecisionMillis: Long,
-        private val original: TestCoroutineDispatcher = TestCoroutineDispatcher()
-    ) : CoroutineDispatcher(), Delay, DelayController by original {
-        override fun dispatch(context: CoroutineContext, block: Runnable) {
-            original.dispatch(context, block)
-        }
-
-        override fun scheduleResumeAfterDelay(timeMillis: Long, continuation: CancellableContinuation<Unit>) {
-            val delay = ceil(timeMillis.toDouble() / delayPrecisionMillis).toInt() * delayPrecisionMillis
-            original.scheduleResumeAfterDelay(delay, continuation)
-        }
-    }
-}
\ No newline at end of file
+//package org.jetbrains.skiko
```

Comment:
these are JVM tests. Do they need to be commented?

There're 2 more:
skiko/src/jvmTest/kotlin/org/jetbrains/skiko/RendezvousBroadcastChannelTest.kt
skiko/src/jvmTest/kotlin/org/jetbrains/skiko/TaskTest.kt

Reply:
Ooooops.....sorry....redundant change.